### DISCUSSION TITLE
How to draw structure with correct S-group brackets

### DISCUSSION THREAD

**Discussion by ihuang8** [epam/ketcher#4682] [Q&A]
I have a molecule that looks like this: 
<img width="266" alt="image" src="https://github.com/epam/ketcher/assets/78018775/6662b63e-8a2f-4bf2-85f1-6c5992320415">

How can I build this in ketcher? I can't seem to get my SRU brackets to be aligned properly. When I try to add the SRU brackets, I get the following: 

<img width="260" alt="image" src="https://github.com/epam/ketcher/assets/78018775/84b404d1-cced-40c3-ad3a-99077518d8c6">

**Comment by flange-ipb** [comment]
Hi @ihuang8,

(1) you also need to have the polymer side chain inside the S-Group selection and (2) be very careful in the selection of the backbone atoms and bonds to be included in the S-Group in order to get the correct star atom arrangement.

Please see my screenshot on how to place the selection box for the S-Group (Ketcher v2.20.0):
![S-Group_Polymer](https://github.com/epam/ketcher/assets/34936695/d7359651-a03d-4259-9544-3eab522bce0c)

The resulting SRU polymer is:
![S-Group_Polymer2](https://github.com/epam/ketcher/assets/34936695/8bed379a-75cf-4f82-9e02-066ea974400c)
(I'd say the hydrogens on the star atoms are a bug.)

After a structure cleanup and a bit of rotation (hold CTRL to get finer rotation intervals) I get:
![S-Group_Polymer3 png](https://github.com/epam/ketcher/assets/34936695/bc7c61f6-2178-4ddd-9404-780658fe157a)

Now, one last objection could be that the methylene group in the backbone is drawn on the wrong side. I think this could be resolved by rotating the initial structure a bit in order to have the correct items inside the S-Group selection box. 

To be honest: Drawing SRU polymers with star atoms has been much easier in earlier Ketcher versions.
